src/main/java/com/vocabsvc/wordtrainer/controller/WordController.java

```diff
@@ -1,13 +1,43 @@
 package com.vocabsvc.wordtrainer.controller;
 
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.RestController;
+import com.vocabsvc.wordtrainer.model.Word;
+import com.vocabsvc.wordtrainer.service.WordService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
 
 @RestController
 public class WordController {
 
+    @Autowired
+    WordService wordService;
+
     @GetMapping("/words")
-    public String getAllWords() {
-        return "init Done! lets begin";
+    public ResponseEntity<List<Word>> getAllWords() {
+        return new ResponseEntity<>(wordService.getAllWords(), HttpStatus.ACCEPTED);
```

Comment:
for GET, DELETE change the HTTP response from `ACCEPTED` to `OK`
Refer this https://developer.mozilla.org/en-US/docs/Web/HTTP/Status#:~:text=Successful%20responses,-200%20OK&text=The%20meaning%20of%20the%20success,transmitted%20in%20the%20message%20body.